mage_integrations/mage_integrations/destinations/google_cloud_storage/README.md

```diff
@@ -0,0 +1,19 @@
+# Google Cloud Storage
+
+![]()
```

Comment:
Could you find an image for the Google Cloud Storage and add here?

Reply:
I looked for an image, but I didn't find it